sdks/python/tox.ini

```diff
@@ -162,6 +162,12 @@ deps =
   pylint==2.17.5
   isort==4.2.15
   flake8==4.0.1
+# https://github.com/grpc/grpc/issues/37660: ignore pip check
```

Comment:
@tvalentyn let us skip `pip check` for lint.

Reply:
why was this necessary to skip lint - is it directly related to grpc change?

Reply:
nvm, it's https://github.com/grpc/grpc/issues/37660

Reply:
sgtm